pipelines/src/test/groovy/RepoHandlerTest.groovy

```diff
@@ -23,6 +26,8 @@ class RepoHandlerTest {
         Assertions.assertEquals(adoptJson.repository.build_branch, 'master')
         Assertions.assertEquals(adoptJson.repository.pipeline_url, 'https://github.com/adoptium/ci-jenkins-pipelines.git')
         Assertions.assertEquals(adoptJson.repository.pipeline_branch, 'master')
+        // TODO: enable below check when code has merged to "master" , refer to file "pipelines/defaults.json"
```

Comment:
Need to complete TODO?

Reply:
@karianna these unit tests run against the mast defaults json: https://github.com/adoptium/jenkins-helper/blob/69c50f5d31ab75c0ed5582e698838ca54cce9407/src/common/RepoHandler.groovy#L24
So will not succeed for new helper_ref, until this PR is merged